src/System.Private.ServiceModel/tests/Common/Unit/MockEndpointBehavior.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Reviewers: this is the same mock endpoint added in #1602 for IInteractiveChannelInitializer.  It was needed for this PR too.  Whichever PR gets merged 2nd will remove the redundant copy.